hail/src/main/scala/is/hail/expr/types/physical/PBaseStruct.scala

```diff
@@ -149,12 +149,20 @@ abstract class PBaseStruct extends PType {
 
   def allocate(region: Code[Region]): Code[Long] = region.allocate(alignment, byteSize)
 
-  def setAllMissing(off: Code[Long]): Code[Unit] = {
+  def initialize(structAddress: Long, setMissing: Boolean = false): Unit = {
```

Comment:
will be used in copy

Reply:
Can add test if desired. Will be tested as part of other struct tests in the copy PR.